python/Ganga/Runtime/bootstrap.py

```diff
@@ -181,6 +181,9 @@ def manualExportToGPI(my_interface=None):
     from Ganga.GPIDev.Lib.Registry.JobRegistry import jobSlice
     exportToInterface(my_interface, "jobSlice", jobSlice, "Functions")
 
+    import datetime
+    exportToInterface(my_interface, "datetime", datetime, 'Classes')
```

Comment:
Because the JobTime does horrible things, see the comment in VStreamer if the VStreamer eval uses the GPI (which is more sensible) we need the datetime due to the JobTime being stored in a probably bad way. I should have added a line here to say why...